AppController/djinn.rb

```diff
@@ -983,13 +983,15 @@ def status(secret)
         next if app_name == "none"
         stats_str << "    Information for application: #{app_name}\n"
         stats_str << "        Language            : "
-        unless @app_info_map[app_name]['language'].nil?
-          stats_str << "#{@app_info_map[app_name]['language']}\n"
-        else
+        if @app_info_map[app_name]['language'].nil?
```

Comment:
Is this the best check to ensure app isn't running? Can we instead use just @app_info_map[app_name]?